infrastructure/kctf/samples/apache-php/challenge/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM kctf-nsjail
+
+RUN apt-get -y update
+RUN apt-get -y upgrade
+
+RUN ln -s /secrets/flag /chroot/flag
+
+VOLUME /tmp
+VOLUME /var/log/apache2
+VOLUME /var/run/apache2
+
+RUN apt-get install -y php-cgi
+RUN apt-get install -y apache2
+
+RUN service apache2 start
+
+COPY html/ /chroot/var/www/html/
+RUN chmod -R 0755 /chroot/var/www/html
+RUN rm -rf /var/www
+RUN ln -s /chroot/var/www /var/www
+
+RUN mkdir -p /chroot/usr/lib/cgi-bin/
+RUN cp /usr/lib/cgi-bin/php /chroot/usr/lib/cgi-bin/php
+RUN ldd /usr/lib/cgi-bin/php | cut -d' ' -f 3 | xargs -r -i cp {} /chroot{}
```

Comment:
this is necessary because the default chroot doesnt have the dynamic libraries that php needs